app/com/arpnetworking/metrics/portal/scheduling/JobCoordinator.java

```diff
@@ -133,10 +153,12 @@ public void preStart() throws Exception {
                 .build();
     }
 
+    // CHECKSTYLE.OFF: ParameterNumber - This is only called in one location.
```

Comment:
@speezepearson is there a reason this whole method is static? If this were not static, I could refactor all of this by just passing a method reference `this::runAntiEntropy` as the runnable rather than the returned value from `createAntiEntopyRunnable`

I'm guessing it might have to do with the actors possibly running on different hosts?

Reply:
I think you can just make this an instance method. I believe I was under the mistaken impression that a periodic timer's messages would pile up, if the actor took more than 1 period to action them.